RemnantSaveGuardian/game.json

```diff
@@ -510,6 +512,54 @@
         }
       ]
     },
+    "DLC1": {
+      "Ring_WoodRing": [
+        {
+          "name": "/Items/Trinkets/Rings/WoodRing/Ring_WoodRing",
+          "notes": "Can be found as a random drop in Losomn."
+        }
+      ],
+      "Weapon_AbyssalHook": {
```

Comment:
Wrong type here - this should be an array of objects, like you did for Wood Ring.